docs/strategy_analysis_example.md

```diff
@@ -7,18 +7,21 @@ Debugging a strategy can be time-consuming. FreqTrade offers helper functions to
 
```

Comment:
```suggestion
The following assumes you work with SampleStrategy, data for 5m timeframe from Binance and have downloaded them into the data directory in the default location.

```

Pls also change FreqTrade --> Freqtrade above, cannot put the comment there since it's not been changed in the PR.

Reply:
yeah - didn't see that - sorry